docs/config_entries_config_flow_handler.md

```diff
@@ -249,6 +250,26 @@ async def async_migrate_entry(hass, config_entry: ConfigEntry):
     return True
 ```
 
+## Reconfigure
```

Comment:
We should make sure the difference between reconfigure and reauthenticate is clear.

Reply:
That whole page needs a bit of an overhaul but let's do that in a follow-up 👍